### ISSUE TITLE
Support MongoDB as a database

### ISSUE THREAD

**Issue by nlaz** [MLH/mlh-hackathon-nodejs-starter#16]
### Description

This starter kit supports a typical SQL-type database using PostgresSQL and [Sequelize](https://github.com/sequelize/sequelize). We did this because relational databases can be more reliable and can have less gotchas (I know this is a matter of opinion). However, it might be prudent to support a NoSQL-type database like MongoDB because it can be quicker to build a hackathon project with. A hackathon project can benefit from flexible data schemas using MongoDB. MongoDB also has a bunch of documentation to help beginners through using it.

To integrate MongoDB into this project, it might be smart to use a library like [mongoose](https://www.npmjs.com/package/mongoose) to help with object querying.

I wouldn't like to simply replace our PostgresSQL/Sequelize code with MongoDB/mongoose code. Ideally, people can choose from one or the other. If this requires setting up a separate repo, that is possible or we can create a script for creating a example project similar to `[create-react-app](https://github.com/facebook/create-react-app)` or `[create-next-app](https://github.com/vercel/next.js/tree/canary/examples/basic-export#using-create-next-app)`.

Additionally, it would be great to add documentation around what is MongoDB, how to use it,  and how to get it running locally and how to deploy.

**Comment by Athira2199** [comment]
I would like to contribute in the mongodb version. Is the project still going on ?

**Comment by Imma-Olly** [comment]
Yes it is still going on. Give me a couple hours to upload the changes then I’ll send you an invite.

**Comment by Athira2199** [comment]
That would be great.
